### PR TITLE
Docs for experiments api 

### PR DESCRIPTION
**Pull request by andrewschreiber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,79 +1,134 @@
 # Hyperdash Python SDK
 
-The Hyperdash Python SDK is the official SDK for [Hyperdash.io](https://hyperdash.io). Once installed, the SDK automatically monitors your machine learning jobs.
+[Hyperdash](https://hyperdash.io) is a machine learning monitoring library, written in Python and capable of running alongside Tensorflow, Scikit-Learn, and other modelling libraries. Hyperdash provides visualizations similar to Tensorboard. It was developed with a focus on enabling fast knowledge gain.
```
Comment (richardartoul) [inline README.md]: I don't really like saying we provide visualizations similar to tensorflow yet cause we don't, and even if we did it would be the web / mobile U.I's doing it not the SDK, but maybe its fine since it gets the point across?

File: README.md [hunk 2]
```diff
@@ -1,79 +1,134 @@
...
 
-## Installation
+Use Hyperdash if you need model monitoring that:
 
-`pip install --upgrade pip && pip install hyperdash`
+* Is fast and easy to setup and use.
+* Tracks your hyperparmeters across different model experiments.
```
Comment (richardartoul) [inline README.md]: typo on hyperparameters

File: README.md [hunk 3]
```diff
@@ -1,79 +1,134 @@
...
+* Graphs performance metrics (loss, reward, etc.) in real-time.
+* Saves a log file of your experiment job output.
+* Notifies you when a long-running experiment job has finished.
+* Can be viewed remotely via both a web and mobile app.
```
Comment (richardartoul) [inline README.md]: "Can be viewed remotely on mobile and web." is cleaner

File: README.md [hunk 4]
```diff
@@ -1,79 +1,134 @@
...
 
-## Installation
+Use Hyperdash if you need model monitoring that:
 
-`pip install --upgrade pip && pip install hyperdash`
+* Is fast and easy to setup and use.
+* Tracks your hyperparmeters across different model experiments.
+* Graphs performance metrics (loss, reward, etc.) in real-time.
+* Saves a log file of your experiment job output.
+* Notifies you when a long-running experiment job has finished.
```
Comment (richardartoul) [inline README.md]: remove "job" or make it "experiment / job"

File: README.md [hunk 5]
```diff
@@ -1,79 +1,134 @@
...
+[Hyperdash](https://hyperdash.io) is a machine learning monitoring library, written in Python and capable of running alongside Tensorflow, Scikit-Learn, and other modelling libraries. Hyperdash provides visualizations similar to Tensorboard. It was developed with a focus on enabling fast knowledge gain.
 
-## Installation
+Use Hyperdash if you need model monitoring that:
 
-`pip install --upgrade pip && pip install hyperdash`
+* Is fast and easy to setup and use.
+* Tracks your hyperparmeters across different model experiments.
+* Graphs performance metrics (loss, reward, etc.) in real-time.
+* Saves a log file of your experiment job output.
```
Comment (richardartoul) [inline README.md]: "Saves your experiment's output (standard out / error) as a local log file"

File: README.md [hunk 6]
```diff
@@ -1,79 +1,134 @@
...
-The Hyperdash Python SDK is the official SDK for [Hyperdash.io](https://hyperdash.io). Once installed, the SDK automatically monitors your machine learning jobs.
+[Hyperdash](https://hyperdash.io) is a machine learning monitoring library, written in Python and capable of running alongside Tensorflow, Scikit-Learn, and other modelling libraries. Hyperdash provides visualizations similar to Tensorboard. It was developed with a focus on enabling fast knowledge gain.
 
-## Installation
+Use Hyperdash if you need model monitoring that:
 
-`pip install --upgrade pip && pip install hyperdash`
+* Is fast and easy to setup and use.
+* Tracks your hyperparmeters across different model experiments.
+* Graphs performance metrics (loss, reward, etc.) in real-time.
```
Comment (richardartoul) [inline README.md]: Should we say tracks instead?

File: README.md [hunk 7]
```diff
@@ -1,79 +1,134 @@
...
-The easiest way to use the Hyperdash SDK is to simply prefix any terminal command with `hyperdash run`:
+# Or signup free with an email
+$ hyperdash signup
+```
+After `login` or `signup`, an API key is saved to your local machine for automatic authorization. If you'd rather manage your API key manually, then review the "API Key Storage" section below.
 
+You're ready to use Hyperdash! Make sure Hyperdash works by running:
 ```
-hyperdash run -n "My test python script" python my_test_script.py
+$ hyperdash demo
```
Comment (richardartoul) [inline README.md]: Might as well update the script in the demo command so that it generates hyperparameters and metrics

File: README.md [hunk 8]
```diff
@@ -1,79 +1,134 @@
...
 ```
 
-or
+## Learn Hyperdash in 30 seconds
```
Comment (richardartoul) [inline README.md]: I think this intro needs a little work. We have three ways of using Hyperdash (not including Jupyter) and we should probably introduce them in order of increasing complexity so we don't lose users in the funnel.

I.E I would start off by saying, if you just want to know when your job is complete and view your logs remotely, use the hyperdash run command blah blah blah

Alternatively, you can decorate the main function in your python script with the monitor decorator blah blah blah

Finally, we have an experiment API which blah blah blah

Comment (richardartoul) [reply README.md]: Basically get it working with their code as fast as possible, then teach them how to do more complicated stuff once they've seen how easy it is to get the basics

File: README.md [hunk 9]
```diff
@@ -1,79 +1,134 @@
...
+{ gamma     : 0.001  }
+| accuracy  : 1.000  |
+Logs for this run of Digits Classifier are available locally at: /Users/username/.hyperdash/logs/digits-classifier/digits-classifier_2017-09-20t18-50-55-258215.log
+```
+**Done!**
+
+Now you can visualize your experiments in the Hyperdash [__web__](https://hyperdash.io/dashboard), [__iOS__](https://itunes.apple.com/us/app/hyperdash-machine-learning-monitoring/id1257582233), and [__Android__](https://play.google.com/store/apps/details?id=com.hyperdash) apps.
 
-Once you've imported our library, run your program as you normally would:
+### IPython/Jupyter Notebook
```
Comment (richardartoul) [inline README.md]: Same comment as before. Tell them if they just want to track a cell really quickly, add this one line of code, but alternatively both the decorator and experiment API also work in Jupyter

File: README.md [hunk 10]
```diff
@@ -1,79 +1,134 @@
...
+Note: by default all print statements will be redirected to the cell that creates the experiment object due to capturing Jupyter's stdio. Use `exp = Experiment("model name", capture_io=False)` for normal printing, but no logging.
 
-### IPython/Jupyter Notebook
+It's also important to `end()` your experiment. Please do so to avoid bugs.
+
+
+### Decorating a Python function
+Import the monitor function, and apply it as a decorator to a function that runs your machine learning job. The only argument you need to pass to the monitor function is the name of the model that you're training.
+
+```
```
Comment (richardartoul) [inline README.md]:
```python

File: README.md [hunk 11]
```diff
@@ -1,79 +1,134 @@
...
+from hyperdash.sdk import monitor
 
-Hyperdash works in IPython/Jupyter notebooks as well. In fact, you can use the exact same code from the previous section in a Jupyter notebook and it will work just fine.
+@monitor("dogs vs. cats")
+def train_dogs_vs_cats(hd_client): # Get hd_client Experiment object as argument to function.
```
Comment (richardartoul) [inline README.md]: A one or two line explanation of the hd_client would be good

File: README.md (hunk 1 above)
Comment (andrewschreiber) [reply README.md]: We are conceptually moving towards Tensorboards visualizations, though we have a lot of distance to cover. I think it's reasonable to state the direction of work, as long as we're candid about where we are (via screenshots, examples, etc.) I'll play with the wording to make that more clear.

File: README.md (hunk 6 above)
Comment (andrewschreiber) [reply README.md]: I think it's implied, and I'd rather focus on the visual of real-time graphs.

File: README.md (hunk 7 above)
Comment (andrewschreiber) [reply README.md]: Agreed.

File: README.md [hunk 12]
```diff
@@ -1,89 +1,134 @@
 # Hyperdash Python SDK
 
-The Hyperdash Python SDK is the official SDK for [Hyperdash.io](https://hyperdash.io). Once installed, the SDK automatically monitors your machine learning jobs.
+[Hyperdash](https://hyperdash.io) is a machine learning monitoring library capable of running alongside Tensorflow, Scikit-Learn, and other modelling libraries. It was developed with a focus on enabling fast knowledge gain.
```
Comment (spence) [inline README.md]: Modeling in the US has a single lowercase L.

File: README.md [hunk 13]
```diff
@@ -1,89 +1,134 @@
...
+$ hyperdash demo
 ```
 
-or
-
+# Learn Hyperdash in 60 seconds
+## Pure logging 
+If all you need is logging and notifications, simply prefix **any** terminal command:
+```bash
+hd run -n "Hotdog CNN" python hotdog.py
```
Comment (spence) [inline README.md]: Are we jumping right to hd? Why are both referenced? Let’s pick one and stick with it.

Comment (andrewschreiber) [reply README.md]: Yeah let's jump to `hd`

File: README.md (hunk 12 above)
Comment (andrewschreiber) [reply README.md]: Murica

File: README.md [hunk 14]
```diff
@@ -23,16 +23,16 @@ $ pip install --upgrade pip && pip install hyperdash
 Installing within a python virtual environment such as [virtualenv](https://github.com/pypa/virtualenv) is recommended. If you are having trouble installing via pip, a virtual environment will usually fix the problem.
```
Comment (spence) [inline README.md]: We may want to explicitly state that users get `hd` when they install `hyperdash`. Additionally, existing users are going to wonder why the change.

Comment (andrewschreiber) [reply README.md]: Yeah I can make that more explicit. Also 'hyperdash' will still work
